index.js

```diff
@@ -805,14 +807,16 @@ Cartero.prototype.writeMetaDataFile = function( callback ) {
 	}, {} );
 
 	var metaData = JSON.stringify( {
-		formatVersion : 3,
+		formatVersion : 4,
 		packageMap : packageMap,
 		entryPointMap : entryPointMap,
+		entryPoints : _this.entryPoints,
```

Comment:
Hi @pepoviola we going to call this something else... no? Like a big long name, I can't remember what tho...